### PR TITLE
fix(rbac): update rbac-backend plugin to version 1.6.3

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
## Description

Update rbac-backend plugin to version 1.6.2 to make plugin working with ssl.

## Which issue(s) does this PR fix

- Fixes janus-idp/backstage-plugins#913

- ## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: .changeset/slimy-numbers-grab.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'backend': major
```
Comment (kadel) [inline .changeset/slimy-numbers-grab.md:2]: not sure about `major` here. It is just a bug fix that is not breaking any compatibility, should it be just a `patch`?

Comment (AndrienkoAleksandr) [reply .changeset/slimy-numbers-grab.md]: Ok, done.

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Review by invincibleJai — APPROVED** [review]
LGTM

**Comment by AndrienkoAleksandr** [comment]
Looks like I need collect on more fix from this pr:  https://github.com/janus-idp/backstage-plugins/pull/929

**Pull request merged**